proto/info/info.proto

```diff
@@ -71,7 +71,6 @@ enum ZInfoTypes {
   ZiEdgeview = 12;
   ZiLocation = 13;
   ZiPatchEnvelope = 14;
-  ZiOpaqueAppInstanceStatus = 15;
```

Comment:
Sorry I missed this one. Makes sense to put a // deprecated = 14;
comment here as well.